cmd/init.go

```diff
@@ -163,25 +163,14 @@ func initAppsody(stack string, template string, config *initCommandConfig) error
 
 		config.Debug.log("Attempting to locate stack ", projectType, " in repo ", repoName)
 		index = indices[repoName]
-		projectFound := false
 		stackFound := false
 		var stackReqs StackRequirement
 
-		// we should remove this as we no longer have use for the v1 index and it is bringing down our code coverage
 		if strings.Compare(index.APIVersion, supportedIndexAPIVersion) == 1 {
```

Comment:
I thought that the `Compare` block checks that the users index has a greater APIVersion than what we support rather than one thats less, hence why I added the extra check.